src/components/infoModalContents/ApprovalInfoContent.tsx

```diff
@@ -10,9 +10,9 @@ export const ApprovalInfoContent = () => {
       <Typography>
         <Trans>
           Before supplying, you need to approve its usage by the Aave protocol. You can learn more
-          in our
+          in our F
```

Comment:
that's for sure wrong as now only AQ is linked :sweat_smile: 

Instead you can just add a space as `{" "}`

Reply:
aye aye captain